addons/tidb/scripts/pd_start.sh

```diff
@@ -29,6 +29,12 @@ if [[ -f $DATA_DIR/join ]]; then
 elif [[ ! -d $DATA_DIR/member/wal ]]; then
     echo "first started pod"
     replicas=$(echo "${KB_POD_LIST}" | tr ',' '\n')
+    # FIXME: Relying on leader status to determine whether to join or initialize a cluster 
```

Comment:
KB can render a list of envs KB_CLUSTER_OPERATION/KB_COMPONENT_OPERATION/KB_POD_OPERATION to Pod to tell the current operation. It helps to identify the operation types and phase of a cluster/component/pod.

Reply:
When the Pod is doing another operation, KB rerender it to another operation type.

Reply:
There is no such env.

Reply:
And environment variables set through envFrom do not automatically update when the ConfigMap is modified. Therefore, unless the Pod is restarted, the env variables will not change dynamically.

Reply:
What I need here is to distinguish whether the pod is started in cluster initialization or in cluster hscale process. Because these two circumstances will map to different start args.

Reply:
> There is no such env.

KB can import such envs to tell if pod is in creating or in hscale/restart/... process, the operator will know it, so it can render it, so the scripts will get the operation context.